openretina/misc.py

```diff
@@ -44,6 +44,10 @@ def set_seed(seed=None, seed_torch=True):
     return seed
 
 
+class MaxLinesExceeded(Exception):
```

Comment:
```suggestion
class MaxLinesExceededException(Exception):
```
This makes it a bit more clear when dealing with the exception later